src/Limelight.h

```diff
@@ -485,17 +491,23 @@ typedef struct _CONNECTION_LISTENER_CALLBACKS {
 void LiInitializeConnectionCallbacks(PCONNECTION_LISTENER_CALLBACKS clCallbacks);
 
 // ServerCodecModeSupport values
-#define SCM_H264        0x00001
-#define SCM_HEVC        0x00100
-#define SCM_HEVC_MAIN10 0x00200
-#define SCM_AV1_MAIN8   0x10000 // Sunshine extension
-#define SCM_AV1_MAIN10  0x20000 // Sunshine extension
+#define SCM_H264            0x00001
+#define SCM_H264_HIGH8_444  0x00004 // Sunshine extension
```

Comment:
This one is inside the range of values that GFE can also use. I'm not sure if they are actually ever setting that bit, but let's use a value above 0xFFFF to be safe

Reply:
Moved the new H.264 bit to 0x00010000
The other new bits should be fine, right?

Reply:
Wait, I didn't. This is SCM not VIDEO_FORMAT....

Reply:
Moved all new SCM flags above 0xFFFF, in case GFE unexpectedly announce them. 9 bits remain unused (for 32-int), it should be enough for the potential yuv444_in_yuv420 formats and a potential new video codec,